src/main/java/uk/gov/hmcts/reform/bulkscan/factory/BulkScanServiceFactory.java

```diff
@@ -22,6 +22,9 @@ private BulkScanServiceFactory(List<BulkScanService> services) {
     }
 
     public static BulkScanService getService(FormType caseType) {
+        if (caseType.equals(FormType.A58_STEP_PARENT) || caseType.equals(FormType.A58_RELINQUISHED)) {
+            return bulkScanServiceCache.get(FormType.A58);
```

Comment:
are we again setting FormType as A58 again ?

Reply:
its just for taking service class. we are having A58service class only.